brainscore/metrics/spatial_correlation.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from brainscore.metrics import Score, Metric
+
+
+def _score(scores):
+    center = np.mean(scores)
+    error = np.std(scores)
+    score = Score([center, error], coords={'aggregation': ['center', 'error']}, dims=('aggregation',))
+    score.attrs[Score.RAW_VALUES_KEY] = scores
+
+    return score
+
+
+class SpatialCorrelationSimilarity(Metric):
+
+    def __init__(self, metric, bin_size_mm):
+        '''
+        Computes the similarity of two given distributions using a given metric. The metric is applied to each bin which
+        in turn are created based on a given bin size and the independent variable of the distributions
+
+        :param metric: metric to be applied to each bin
+        :param bin_size_mm: size per bin in mm | one fixed size, utilize Score.RAW_VALUES_KEY to change weighting
+        '''
+        self._metric = metric
+        self._bin_size = bin_size_mm
+
+    def __call__(self, target_statistic, candidate_statistic):
+        '''
+        :param target_statistic: list of 2 lists, [0] holds the independent variable -> binning over this
+        :param candidate_statistic: list of 2 lists, [0] holds the independent variable -> binning over this

```

Comment:
needs more explanation of what is in the lists

Reply:
How about: `list of 2 lists, [0] distances -> binning over this, [1] correlation per distance value`